site/content/docs/5.3/about/support-bootstrap.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: docs
+title: Support Bootstrap
+description: Show your support through donations and sponsorships to help maintaining Bootstrap
+group: about
+toc: true
+---
+
+Through donations and sponsorships we are able to maintain & improve Bootstrap. Feel free to show your support on our [Open Collective page](https://opencollective.com/bootstrap).
+
+## Sponsors
+
+{{< sponsors.inline >}}
+<div id="sponsorList" class="d-flex flex-wrap gap-3 text-center mt-3"></div>
+{{< /sponsors.inline >}}
+
+## Backers
+
+{{< backers.inline >}}
+<div id="backerList" class="d-flex flex-wrap gap-3 text-center mt-3"></div>
+{{< /backers.inline >}}
+
+## Services
+
+{{< services.inline >}}
+<div class="d-flex flex-wrap gap-3 mt-3">
+  {{- range (index $.Site.Data "services") }}
+    <div class="position-relative">
+      {{ if .website -}}
+        <a href="{{ .website }}" class="stretched-link text-reset" title="{{ .name }}" target="_blank" rel="nofollow noopener noreferrer">
+      {{ end -}}
+      <img src="{{ printf ("/docs/%s/assets/img/services/%s") $.Site.Params.docs_version .image }}" alt="{{ .name }}" class="mh-100 mw-100">
```

Comment:
Since they are static images, we have an issue with dark mode, maybe add a static background to its parent or use the svg directly (this way we can color the text using `var(--bs-body-color)` or something similar) ?